app/core/Router.coffee

```diff
@@ -72,15 +72,15 @@ module.exports = class CocoRouter extends Backbone.Router
     'contribute/diplomat': go('contribute/DiplomatView')
     'contribute/scribe': go('contribute/ScribeView')
 
-    'courses': go('courses/CoursesView')
-    'Courses': go('courses/CoursesView')
-    'courses/students': redirect('/courses')
+    'courses': redirect('/students') # Redirected 9/3/16
+    'Courses': redirect('/students') # Redirected 9/3/16
```

Comment:
Yikes, good catch on the redirects.
You can join a class and become a student with a query param, so I think that's why we delay showing an error screen there.
https://github.com/codecombat/codecombat/blob/students-routing/app/views/courses/CoursesView.coffee#L170